lightbulb/slash_commands/context.py

```diff
@@ -187,20 +187,18 @@ async def respond(self, content: hikari.UndefinedType = hikari.UNDEFINED, **kwar
         Returns:
             ``None``
 
-        Raises:
-            :obj:`hikari.NotFoundError`: The interaction has already had an initial response sent for it. This is
-                determined without sending a REST request so you are safe to try except this in your code.
-
         Note:
-            This can only be called **once** for each interaction. To add more information to the response
-            you should use :obj:`~lightbulb.slash_commands.SlashCommandContext.edit_response`
+            This will be a shortcut to :obj:`~lightbulb.slash_commands.SlashCommandContext.edit_response`
+            if you have called this method once.
```

Comment:
I recommend `if you have already responded to this interaction.` as they may call respond many times and this only explicitly states once. This also clarifies that this only applies to this particular interaction.

Reply:
Good point.